bagitobjecttransfer/recordtransfer/context_processors.py

```diff
@@ -1,5 +1,5 @@
 from recordtransfer.settings import SIGN_UP_ENABLED, MAX_SINGLE_UPLOAD_SIZE, \
-    MAX_TOTAL_UPLOAD_COUNT, MAX_TOTAL_UPLOAD_SIZE, FILE_UPLOAD_ENABLED
+    MAX_TOTAL_UPLOAD_COUNT, MAX_TOTAL_UPLOAD_SIZE, FILE_UPLOAD_ENABLED, MODE
```

Comment:
MODE can be removed here

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/248/commits/ee359aacf3ca6f2c42350fbfcbe9de5eb549d3c0

Reply:
Resolved by [Remove redundant MODE setting](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/248/commits/ee359aacf3ca6f2c42350fbfcbe9de5eb549d3c0)